lib/src/rules/directives_ordering.dart

```diff
@@ -249,6 +347,11 @@ class _Visitor extends SimpleAstVisitor {
         .forEach(reportDirective);
   }
 
+  Iterable<ExportDirective> _getExportDirectives(CompilationUnit node) =>
+      node.directives
+          .where(_isExportDirective)
+          .map((e) => e as ExportDirective);
```

Comment:
Do we need this `map` operation? If so, do we also need it in `_getImportDirectives`?